osect_sensor/SPECIFICATION_en.md

```diff
@@ -0,0 +1,467 @@
+
+Communication logs do not contain control information of the production line.
+
+The packets/frames targeted by each file, the information to be recorded, and the generation method are as follows.
+
+
+Packets/frames targeted by each file, recorded information, and generation methods for each file are as follows.
+
+### 2.1. IP connection Log
+
+Description: The log of IP communication connection information
+
+Target: IPv4 and IPv6 packets whose transport layer protocols are TCP, UDP, ICMP, and ICMPv6
+
+Generation method: Zeek
+
+File name: conn.log
+
+Recording Information: the table below
+
+- Zeek official information: <https://docs.zeek.org/en/master/scripts/base/protocols/conn/main.zeek.html#detailed-interface>
+- The host serving as the starting point for the connection is defined as the client, and the communication destination of the client is defined as the server.
+
+| Information Name | Description |
+| --- | --- |
+| ts | timestamp of the first packet of the connection |
+| uid | unique identification of the connection |
+| id.orig_h | client IP address |
+| id.orig_p | client port number (type, code for ICMP and ICMPv6) |
+| id.resp_h | server IP address |
+| id.resp_p | server port number (type, code for ICMP and ICMPv6) |
+| proto | transport layer protocol |
+| service | application layer protocol |
+| duration | connection duration |
+| orig_bytes | number of payload bytes sent by the client |
+| resp_bytes | number of payload bytes sent by the server |
+| conn_state | connection state |
+| local_orig | (no record) |
+| local_resp | (no record) |
+| missed_bytes | number of bytes not delivered due to packet loss |
+| history | connection state history |
+| orig_pkts | number of packets sent by the client |
+| orig_ip_bytes | number of IP-level bytes sent by the client |
+| resp_pkts | number of packets sent by the server |
+| resp_ip_bytes | number of IP-level bytes sent by the server |
+| tunnel_parents | uid value of the encapsulated parent connection used during the lifespan of this internal connection when it is over a tunnel |
+| orig_l2_addr | client MAC address |
+| resp_l2_addr | server MAC address |
+
+
+### 2.2. IP Flow Log
+
+Description: The log of bidirectional flow information of IP communication
+
+Target: IPv4 and IPv6 packets whose transport layer protocols are other than TCP, UDP, ICMP, and ICMPv6
+
+Generation method: generate by the software developed by NTT Com based on data from the open source software YAF
+
+File name: yaf_flow.log
+
+Recording Information: the table below
+
+- YAF official information: <https://tools.netsa.cert.org/yaf/yafscii.html#Human-Readable-Output>
+- The host serving as the starting point for the connection is defined as the client, and the communication destination of the client is defined as the server.
+
+
+| Information Name | Description |
+| --- | --- |
+| ts | start time the flow |
```

Comment:
Here!
It is translated into "start time the flow", because the Japanese version is written as below.
Is it correct?
<img width="873" alt="Screenshot 2024-01-09 at 9 59 24" src="https://github.com/nttcom/OsecT/assets/57517810/aaac0217-446a-4b64-a4e0-3ac7247862b3">